Assets/ConsentManagementProvider/Scripts/json/wrappers/GdprConsentWrapper.cs

```diff
@@ -9,11 +9,7 @@ internal class GdprConsentWrapper
         public Dictionary<string, object> TCData;
         public Dictionary<string, SpVendorGrantWrapper> grants;
         public List<string> acceptedCategories;
-        public bool applies;
         public string webConsentPayload;
         public ConsentStatusWrapper consentStatus;
-#nullable enable
-        public GCMDataWrapper? gcmStatus;
-#nullable disable
     }
```

Comment:
@Nevazhnovu were `applies` an `gcmStatus` removed on purpose?

Reply:
@andresilveirah Yup, that's done purposefully. The `applies` and `gcmStatus` have moved to `SpGdprConsentWrapper<platform>.cs` classes